pkg/render/manager.go

```diff
@@ -1166,3 +993,183 @@ func managerClusterWideDefaultView() *v3.UISettings {
+	rules := []rbacv1.PolicyRule{
+		{
+			APIGroups: []string{"authorization.k8s.io"},
+			Resources: []string{"subjectaccessreviews"},
+			Verbs:     []string{"create"},
+		},
+		{
+			APIGroups: []string{"authentication.k8s.io"},
+			Resources: []string{"tokenreviews"},
+			Verbs:     []string{"create"},
+		},
+		{
+			APIGroups: []string{"projectcalico.org"},
+			Resources: []string{
+				"networksets",
+				"globalnetworksets",
+				"globalnetworkpolicies",
+				"tier.globalnetworkpolicies",
+				"networkpolicies",
+				"tier.networkpolicies",
+				"stagedglobalnetworkpolicies",
+				"tier.stagedglobalnetworkpolicies",
+				"stagednetworkpolicies",
+				"tier.stagednetworkpolicies",
+				"stagedkubernetesnetworkpolicies",
+			},
+			Verbs: []string{"list"},
+		},
+		{
+			APIGroups: []string{"projectcalico.org"},
+			Resources: []string{
+				"stagednetworkpolicies",
+				"tier.stagednetworkpolicies",
+			},
+			Verbs: []string{"patch"},
+		},
+		{
+			APIGroups: []string{"projectcalico.org"},
+			Resources: []string{
+				"tiers",
+			},
+			Verbs: []string{"get", "list"},
+		},
+		{
+			APIGroups: []string{"projectcalico.org"},
+			Resources: []string{
+				"hostendpoints",
+			},
+			Verbs: []string{"list"},
+		},
+		{
+			APIGroups: []string{"projectcalico.org"},
+			Resources: []string{
+				"felixconfigurations",
+			},
+			ResourceNames: []string{
+				"default",
+			},
+			Verbs: []string{"get"},
+		},
+		{
+			APIGroups: []string{"projectcalico.org"},
+			Resources: []string{
+				"alertexceptions",
+			},
+			Verbs: []string{"get", "list", "update"},
+		},
+		{
+			APIGroups: []string{"networking.k8s.io"},
+			Resources: []string{"networkpolicies"},
+			Verbs:     []string{"get", "list"},
+		},
+		{
+			APIGroups: []string{"policy.networking.k8s.io"},
+			Resources: []string{
+				"adminnetworkpolicies",
+				"baselineadminnetworkpolicies",
+			},
+			Verbs: []string{"list"},
+		},
+		{
+			APIGroups: []string{""},
+			Resources: []string{"serviceaccounts", "namespaces", "nodes", "events", "services", "pods"},
+			Verbs:     []string{"list"},
+		},
+		{
+			APIGroups: []string{"apps"},
+			Resources: []string{"replicasets", "statefulsets", "daemonsets"},
+			Verbs:     []string{"list"},
+		},
+		// When a request is made in the manager UI, they are proxied through the Voltron backend server. If the
+		// request is targeting a k8s api or when it is targeting a managed cluster, Voltron will authenticate the
+		// user based on the auth header and then impersonate the user.
+		{
+			APIGroups: []string{""},
+			Resources: []string{"users", "groups", "serviceaccounts"},
+			Verbs:     []string{"impersonate"},
+		},
+		// Allow query server talk to Prometheus via the manager user.
+		{
+			APIGroups: []string{""},
+			Resources: []string{"services/proxy"},
+			ResourceNames: []string{
+				"https:tigera-api:8080", "calico-node-prometheus:9090",
+			},
+			Verbs: []string{"get", "create"},
+		},
+		{
+			// Add access to Linseed APIs.
```

Comment:
I don't think these are needed for managed clusters? There's no linseed in a managed cluster.